internal/bminventory/inventory.go

```diff
@@ -656,6 +656,12 @@ func (b *bareMetalInventory) UpdateCluster(ctx context.Context, params installer
 		tx.Rollback()
 		return installer.NewUpdateClusterNotFound().WithPayload(common.GenerateError(http.StatusNotFound, err))
 	}
+	if b.clusterApi.IsClusterInstallationInProgress(&cluster) {
```

Comment:
```suggestion
	if b.clusterApi.HasClusterInstallationStarted(&cluster) {
```

And check if it's installing-in-progress, installing, installed, or error.  User shouldn't be allowed to change the cluster in these states.

Reply:
from the ticket, I understand that the only in installing state nothing can be updated, no ?

Reply:
@oshercc What @avishayt said :)